code/game/objects/items/holy_weapons.dm

```diff
@@ -390,8 +392,29 @@
 	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
 	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	desc = "Holding this makes you look absolutely devilish."
 	attack_verb = list("poked", "impaled", "pierced", "jabbed")
+	slot_flags = SLOT_BACK
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	sharpness = IS_SHARP
+	force = 10
+	bonus_burn = 5
+	
+/obj/item/nullrod/pitchfork/clockwork_spear
+	icon_state = "ratvarian_spear"
+	lefthand_file = 'icons/mob/inhands/antag/clockwork_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/clockwork_righthand.dmi'
+	name = "clockwork spear"
+	desc = "A pointy spear made of holy brass. It ticks and tocks."
+	attack_verb = list("ticked", "tocked", "singed", "speared")
+	clockwork_desc = "A disgusting and cheap replica created by the church to imitate the divine armory of the Justiciar."
+	hitsound = 'sound/magic/clockwork/ratvar_attack.ogg'
+	
+/obj/item/nullrod/attack(mob/living/target, mob/living/carbon/human/user)
+	. = ..()
+	if(!QDELETED(target) && target.stat != DEAD)
```

Comment:
This entire proc can be done with

```
if(target.stat != DEAD)
    target.adjustFireLoss(bonus_burn)
```

Reply:
This allows us to modify how much burning a particular re-skin does in the future if we need to do so ^.^

I hope my responses don’t come off as defensive, I’ve taken into account your point about the slots, and am fully open to criticism.